### ISSUE TITLE
`force:org:create` ignores `apiVersion` config value when deploying settings

### ISSUE THREAD

**Issue by cristiand391** [forcedotcom/cli#1719]
### Summary

`force:org:create` ignores `apiVersion` config value so it always default to the max api version supported by the org for the settings deployment.

### Steps To Reproduce:

**Repository to reproduce:** [dreamhouse-lwc](https://github.com/dreamhouseapp/dreamhouse-lwc)

1.  Set apiVersion: `sfdx config:set apiVersion=47.0 --global`
2.  run ` DEBUG=sf:deploySettingsViaFolder sfdx force:org:create -s -f config/project-scratch-def.json -a dreamhouse`
3. See output:

```
WARNING: apiVersion configuration overridden at "47.0"
  sf:deploySettingsViaFolder DEBUG deploying to apiVersion: 56.0 +0ms
```

### Expected result

Respect `apiVersion` value

### Actual result

The code always gets `undefined` so it defaults to the max api supported by the org.

### System Information
`sfdx version --verbose --json`

```json
{
  "cliVersion": "sfdx-cli/7.169.1",
  "architecture": "darwin-x64",
  "nodeVersion": "node-v16.16.0",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 1.3.0 (core)",
    "@oclif/plugin-commands 2.2.0 (core)",
    "@oclif/plugin-help 5.1.12 (core)",
    "@oclif/plugin-not-found 2.3.1 (core)",
    "@oclif/plugin-plugins 2.1.0 (core)",
    "@oclif/plugin-update 3.0.0 (core)",
    "@oclif/plugin-version 1.1.2 (core)",
    "@oclif/plugin-warn-if-update-available 2.0.4 (core)",
    "@oclif/plugin-which 2.1.0 (core)",
    "alias 2.1.0 (core)",
    "apex 1.2.0 (core)",
    "auth 2.2.5 (core)",
    "community 2.0.1 (core)",
    "config 1.4.19 (core)",
    "custom-metadata 2.0.0 (core)",
    "data 2.1.2 (core)",
    "generator 2.0.2 (core)",
    "info 2.1.0 (core)",
    "limits 2.0.1 (core)",
    "org 2.2.3 (core)",
    "packaging 1.9.1 (core)",
    "schema 2.1.3 (core)",
    "signups 1.2.0 (core)",
    "source 2.0.13 (core)",
    "telemetry 2.0.0 (core)",
    "templates 55.1.0 (core)",
    "trust 2.0.4 (core)",
    "user 2.1.2 (core)",
    "@salesforce/sfdx-plugin-lwc-test 1.0.1 (core)",
    "salesforce-alm 54.8.1 (core)"
  ],
  "osVersion": "Darwin 20.6.0",
  "shell": "zsh",
  "rootPath": "/Users/cdominguez/.nvm/versions/node/v16.16.0/lib/node_modules/sfdx-cli"
}
```

**Comment by sharad272** [comment]
Hi,
I am also facing this issue while creating scratch orgs. I have made changes in sfdx-project.json but it was of no use.

`error running force:org:create: a scratch org was created with username test-bh8wkklzerdc@example.com, but the settings failed to deploy due to: not available for deploy for this api version`

Can you please help?

**Comment by cristiand391** [comment]
@sharad272 you can try using the `force:org:legacy:create` which should respect the `apiVersion` config. We might have this fixed for the current `force:org:create` command in the RC next week.

**Comment by sharad272** [comment]
> force:org:legacy:creat

@cristiand391  Thanks! This is working for now.

**Comment by shetzel** [comment]
Fixed in CLI version 7.171.0
